blog.html

```diff
@@ -22,6 +22,94 @@
         <!--#include virtual="includes/_nav.htm" -->
         <div class="right">
             <h1 class="content-title">Blog</h1>
+            <article>
+                <h2 class="bullet">
+                    <a id="apache_kafka_380_release_announcement"></a>
+                    <a href="#apache_kafka_380_release_announcement">Apache Kafka 3.8.0 Release Announcement</a>
+                </h2>
+                xx July 2024 - Josep Prat (<a href="https://twitter.com/jlprat">@jlprat</a>)
+                <p>We are proud to announce the release of Apache Kafka 3.8.0. This release contains many new features and improvements. This blog post will highlight some of the more prominent features. For a full list of changes, be sure to check the <a href="https://downloads.apache.org/kafka/3.8.0/RELEASE_NOTES.html">release notes</a>.</p>
+                <p>See the <a href="https://kafka.apache.org/documentation.html#upgrade_3_8_0">Upgrading to 3.8.0 from any version 0.8.x through 3.6.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
+                <p>
+                    In the last release, 3.6,
+                    <a href="https://kafka.apache.org/38/documentation.html#tiered_storage">tiered storage</a> was released as early availability feature.
+                    In this release, Tiered Storage now supports clusters configured with multiple log directories (i.e. JBOD feature).
+                </p>
+                <p>
+                    Up until now, only the default compression level was used by Apache Kafka. From this version on, a configuration mechanism to specify compression level is included. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-390%3A+Support+Compression+Level">KIP-390</a> for more details.
```

Comment:
Shall we add some perf numbers as shown by the tests ran in the KIP? 56.5% better message/s throughput & 40% latency reduction at the cost of 9% more disk usage in the gzip example is definitely worthy of a call out. Even more so the lz4 example. 

Unless I'm missing something, this is a very impactful config for users to change, and it's very easy. We should urge them to explore it

Reply:
Where can I find the results? So I can properly link them in the blog

Reply:
https://cwiki.apache.org/confluence/display/KAFKA/KIP-390%3A+Support+Compression+Level
![image](https://github.com/user-attachments/assets/771eed73-b12e-4f1f-9d50-c6ec5c01be3c)

Reply:
The improvement (or not) really depends on your data. The dataset mentioned in the KIP was in JSON which really benefits from compression.

Reply:
As the entry already mentions "See KIP-390 for more details" I think we are then covered. Unless we want to make the blog post dive deeper into showcasing a particular example of a benchmark of this feature.